index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Matt's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <img
+        class="nameLogo"
+        src="images/matt-knight-logo-new.svg"
+        alt="Matt Knight logo"
+      />
+      <nav>
+        <ul class="nav-container">
+          <li class="nav-item"><a href="#aboutmelink">About Me</a></li>
+          <li class="nav-item"><a href="#projectslink">Projects</a></li>
+          <li class="nav-item"><a href="#footer">Contact</a></li>
+          <li class="nav-item"><a href="images/Matt-Knight-Cv-WebDev.pdf">Resume</a></li>
```

Comment:
Check out the HTML `target` attribute to see how you can get this to open in a new tab - would be useful here so the user isn't navigated away from the page when viewing your CV